src/components/header.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
Exports should always correspond to the file name nso this file should be called Welcome. Please read our Wiki because I'll be very strict during this project.

Reply:
This should also be called App not Welcome